packages/typescript/src/watchProgram.ts

```diff
@@ -3,20 +3,113 @@ import { PluginContext } from 'rollup';
 import { buildDiagnosticReporter } from './diagnostics/emit';
 import { DiagnosticsHost } from './diagnostics/host';
 import { Resolver } from './moduleResolution';
+import { DiagnosticCategory } from 'typescript';
```

Comment:
This doesn't work:
```js 
import('typescript'). DiagnosticCategory
```
`import('typescript')` doesn't have the property `DiagnosticCategory`

`Property 'DiagnosticCategory' does not exist on type 'Promise<{ default: typeof ts; …`